lesson2/zenameless1_hometask2.rb

```diff
@@ -0,0 +1,35 @@
+module M
+
+  def self.included(base)
+    base.include M::InstanceMethods
+    base.extend M::ClassMethods
+    puts "Class methods:\t#{base.singleton_methods}"
+    puts "Instance methods:\t#{base.instance_methods - Object.methods}"
+  end
+
+  module ClassMethods
+
```

Comment:
Why are you so like these empty lines between class/module/method body?
It makes reading more complicated. I wouldn't recommend to use it. At least, in our project.